src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/CSharpTestLspServer.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Nerdbank.Streams;
+using StreamJsonRpc;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.Common
+{
+    public sealed class CSharpTestLspServer : IAsyncDisposable
+    {
+        public readonly AdhocWorkspace TestWorkspace;
+
+        private readonly IRazorLanguageServerTarget _languageServer;
+
+        private readonly StreamJsonRpc.JsonRpc _clientRpc;
```

Comment:
Any reason to not have a using for StreamJsonRpc?

Reply:
The compiler gets confused and emits an error when it's not fully qualified since `JsonRpc` is apparently both a type and namespace ☹️

Reply:
Ah interesting